splunk_connect_for_snmp_poller/manager/hec_sender.py

```diff
@@ -71,7 +70,7 @@ def post_event_data(endpoint, host, variables_binds, index, one_time_flag=False)
 
     try:
         logger.debug(f"+++++++++endpoint+++++++++\n{endpoint}")
-        response = requests.post(url=endpoint, json=data)
+        response = requests.post(url=endpoint, json=data, timeout=60)
```

Comment:
Can't we move "60" to our configuration file? "Magic numbers" are really never a good idea.